schema.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE animals(
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  name VARCHAR(250),
+  date_of_birth DATE,
+  escape_attempts INT,
+  neutered BIT,
+  weight_kg FLOAT
+);
```

Comment:
- [ ] You're expected to add indexes. Adding indexes for FOREIGN KEYS or on columns frequently used in queries is one of the basic solutions for the performance problems